example.py

```diff
@@ -28,20 +30,21 @@ async def cli():
     async with aiohttp.ClientSession() as aiohttp_session, aioshelly.COAP() as coap_context:
         try:
             device = await asyncio.wait_for(
-                aioshelly.Device.create(aiohttp_session, coap_context, options), 5
+                aioshelly.Device.create(aiohttp_session, coap_context, options, init), 5
             )
         except asyncio.TimeoutError:
             print("Timeout connecting to", ip)
             return
 
-        print_device(device)
+        if device.initialized:
+            print_device(device)
```

Comment:
I think that we should always print the device but teach `print_device` to show initializing devices.

Reply:
Fixed